src/util.h

```diff
@@ -517,7 +517,7 @@ struct FunctionDeleter {
 template <typename T, void (*function)(T*)>
 using DeleteFnPtr = typename FunctionDeleter<T, function>::Pointer;
 
-std::set<std::string> ParseCommaSeparatedSet(const std::string& in);
+std::vector<std::string> SplitString(const std::string& in, char delim = ',');
```

Comment:
nit: I think we can do away with the default parameter, as it’s not very obvious from the function name what it is, and it doesn’t take that much work to specify.

Reply:
@TimothyGu yup, done!